grails-app/services/au/org/ala/merit/ProjectService.groovy

```diff
@@ -774,8 +774,8 @@ class ProjectService  {
             // We allow reports to be generated up to 24 hours after the end of a project due to
             // project end dates being 00:00 of the last day of the project instead of 23:59:...
             String plannedEndDateAlignedToReportingSchedule = DateUtils.format(DateUtils.parse(plannedEndDate).plusDays(1))
-            if (lastReport && plannedEndDateAlignedToReportingSchedule < lastReport.toDate) {
-                message = "The project end date must be on or after ${DateUtils.isoToDisplayFormat(lastReport.toDate)}"
+            if (lastReport && plannedEndDateAlignedToReportingSchedule < lastReport.fromDate) {
```

Comment:
This allows the report to be completely deleted.
It should also take into account the minimum report length setting in the project config.